rotkehlchen/chain/ethereum/modules/curve/decoder.py

```diff
@@ -157,26 +181,6 @@ def _decode_curve_deposit_events(
                 event.counterparty = CPT_CURVE
                 event.notes = f'Deposit {event.balance.amount} {crypto_asset.symbol} in curve pool'  # noqa: E501
                 deposit_events.append(event)
-            elif (  # deposit give asset
```

Comment:
It's not needed since the `curve y-deposit` contract addition & removal of liquidity is now being handled at the post decoding level.

`Curve Y-Deposit` contract is similar to `Curve USDN deposit`